tests/test_remote.py

```diff
@@ -505,19 +506,20 @@ def serve_reverse_tunnel(queue):
             queue = Queue()
             tunnel_server = Process(target=serve_reverse_tunnel, args=(queue,))
             tunnel_server.start()
-            message = str(time.time_ns())
-            with rem.tunnel(12222, remote_socket, dhost="localhost", reverse=True):
+            message = str(time.time())
+            with rem.tunnel(12223, remote_socket, dhost="localhost", reverse=True):
                 remote_send_af_inet = """import sys, socket
 s = socket.socket()
 s.connect(("localhost", {}))
 s.send("{}".encode("ascii"))
 s.close()
-""".format(remote_socket, message)
+""".format(
+                    remote_socket, message
+                )
                 (rem.python["-u"] << remote_send_af_inet).popen()
                 tunnel_server.join()
```

Comment:
@oakreid Now this just hangs on macOS Python 3.10. Any idea here? It's probably using a different forking model?

Reply:
Ah, this only works when forking, not when spawning, and the default changed in 3.8+. Skipping on macOS 3.8+ should be fine for now. Ideally, it would be nice to support spawn too.